website/project/metadata/real-world-evidence.json

```diff
@@ -89,25 +89,25 @@
                "type":"string",
                "format":"textarea",
                "title":"Study population",
-               "description":"Please choose one the population under evaluation in the study",
-               "example":"The population under evaluation in the study",
+               "help":"The population under evaluation in the study",
```

Comment:
The JIRA ticket makes it seem like the `help` value should be `Please choose one the population under evaluation in the study.` (maybe that should be `Please choose the population under evaluation in the study.`?)